src/services/control.js

```diff
@@ -305,49 +327,62 @@ var gitPull = function (repoPath, callback) {
 // Read functions
 
 var readContentMap = function (callback) {
-  var contentMapPath = PathService.getConfigPath(config.control_content_file());
+  var contentFiles = globby.sync([
+    PathService.getConfigPath(config.control_content_file()),
+    PathService.getConfigPath('content.d/**/*')
+  ]);
```

Comment:
This will cause weird behavior in the Deconst client. The client takes advantage of the `path.resolve` call in PathService to [override the config files](https://github.com/deconst/client/blob/master/src/utils/ContentRepositoryUtil.js#L419-L420) with doctored versions of certain files from locations outside of the control repo root. Now that we're globbing from a few places, that'd _merge_ the overrides with the existing config rather than replacing it.

I'm... not sure of the best way to work around this, honestly. Maybe push the globby calls up into `PathService` (creating calls into something like `getContentFiles`, `getRouteFiles`, and so on) and only include the `content.d/**/*` globs if `config.control_content_file()` is unset?